src/blocks/__stories__/Alert.stories.tsx

```diff
@@ -39,7 +36,9 @@ export const inverseVariants = () => (
   </div>
 )
 
-export const withCustomIcon = () => (
+export const CustomIcon = () => (
+  // import { Icon } from "@bloom-housing/ui-seeds"
+  // import { HomeModernIcon } from "@heroicons/react/24/solid"
```

Comment:
Do we need these comments here?

Reply:
I have em in there so that they show up in the code sampling in Storybook